ai_edge_torch/debug/test/test_culprit.py

```diff
@@ -48,6 +50,32 @@ def forward(self, x):
     return x
 
 
+class TestSearchModel(unittest.TestCase):
```

Comment:
Please separate this into another file `test_search_model.py`. We usually expect 1 class per test file.